src/frame/js/actions/faye.js

```diff
@@ -155,9 +184,10 @@ export function updateUser(currentAppUser, nextAppUser) {
             disconnectFaye();
 
             return dispatch(subscribeUser()).then(() => {
-                if (nextAppUser.conversationStarted) {
-                    return dispatch(handleConversationUpdated());
-                }
+                // TODO : figure out if still relevant
+                // if (nextAppUser.conversationStarted) {
```

Comment:
this was for the linking for where User A has no conversation yet and links a channel and it gets merged with User B that has a conversation started. The widget would need to "init" itself with the new user state... I'm not sure this flow will still be relevant with the new stuff (or at least will be different).